shopinvader/models/product_product.py

```diff
@@ -59,13 +59,24 @@ def _add_to_cart_allowed(self, backend, partner=None):
             self.shopinvader_bind_ids.filtered(lambda x: x.backend_id == backend)
         )
 
-    def _get_invader_variant(self, backend, lang):
+    def _get_invader_variant(self, backend, lang, only_active=True):
         """Retrieve invader variant by backend and language.
 
         :param backend: backend recordset
         :param lang: lang code
         """
-        return self.shopinvader_bind_ids.filtered(
+        # We perform a first check to include inactive bindings
+        # if either the "only_active" argument is False or
+        # the "active_test" flag from the context is not found.
+        bindings = self.shopinvader_bind_ids
+        active_test = None
+        if "active_test" in self.env.context:
+            active_test = self.env.context.get("active_test")
```

Comment:
```suggestion
        active_test = self.env.context.get("active_test", None)
```

Reply:
default return value for `.get()` is None.
```suggestion
        active_test = self.env.context.get("active_test")
```